packages/enzyme-utilities/README.md

```diff
@@ -56,3 +60,29 @@ describe('<MyComponent />', () => {
   });
 });
 ```
+
+### `function setupMount(): { mount, addMountedWrapper, unmountAllWrappers }
+
+Returns an object containing 3 functions for handling mounting of enzyme wrappers:
+
+* `mount`
+* `unmountAllWrappers`
+* `addMountedWrapper`
+
+These functions are described below.
+
+### `function mount<P, S = any>(node: ReactElement<P>, options?: MountRendererProps | undefined): ReactWrapper<P, S>`
+
+A custom `mount` function mirorring `enzyme`'s `mount` function, designed to assist with automatic tracking of mounted nodes. These can later be unmounted via the `unmountAllWrappers` function described below.
+
+#### `function unmountAllWrappers(): void`
+
+Unmounts all wrappers previously registered via `mount` or `addMountedWrapper`. This can be used in a global `afterEach` hook with the following caveat:
+
+> Usually, you can rely on running this once for the entire suite after each test (and its nested hooks) have been run. However, in some cases you may need to run this manually. This is most commonly needed when your component uses timeouts/ animation frames that you mock out for tests, and waiting until after you have restored those globals will lead to their stored identifiers for the timeout or animation frame to be invalid.
```

Comment:
I sort of feel like the "right" thing to do here is to try to contribute this functionality to enzyme.

Reply:
There is nothing enzyme can do in these cases; it has no way of knowing when to unmount or what parts of the environment may have changed in such a way that unmounting will have negative side effects

Reply:
https://github.com/airbnb/enzyme/issues/1689